apps/spruce/src/pages/Version.tsx

```diff
@@ -18,11 +18,13 @@ import { VERSION } from "gql/queries";
 import { useSpruceConfig } from "hooks";
 import { PageDoesNotExist } from "pages/NotFound";
 import { shortenGithash, githubPRLinkify, jiraLinkify } from "utils/string";
+import { ActionButtons } from "./version/ActionButtons";
 import { WarningBanner, ErrorBanner, IgnoredBanner } from "./version/Banners";
 import VersionPageBreadcrumbs from "./version/Breadcrumbs";
 import BuildVariantCard from "./version/BuildVariantCard";
-import { ActionButtons, Metadata, VersionTabs } from "./version/index";
+import { Metadata } from "./version/Metadata";
 import { NameChangeModal } from "./version/NameChangeModal";
+import VersionTabs from "./version/VersionTabs";
```

Comment:
nit. does the folder name need to be called `version/VersionTabs`? It seems a bit redundant. Could it be `version/tabs`?